diff -Nur dvd+rw-tools-6.1-orig/Makefile.m4 dvd+rw-tools-6.1/Makefile.m4
--- dvd+rw-tools-6.1-orig/Makefile.m4	2006-02-15 23:26:15.259609250 +0100
+++ dvd+rw-tools-6.1/Makefile.m4	2006-02-15 23:47:06.985837250 +0100
@@ -1,7 +1,7 @@
 # OBS! M4 processed!
 changequote([, ])
 [
-CHAIN=growisofs dvd+rw-format dvd+rw-booktype dvd+rw-mediainfo dvd-ram-control
+CHAIN=growisofs dvd+rw-format dvd+rw-booktype dvd+rw-mediainfo dvd-ram-control rpl8 btcflash
 
 dvd+rw-tools:	$(CHAIN)
 
@@ -176,17 +176,17 @@
 manprefix?=$(shell case $(prefix) in (*/usr/?*) echo $(prefix)/man ;; (*) echo $(prefix)/share/man ;; esac)
 
 bin_mode?=0755	# yes, default is *no* set-uid
-minus_o:=$(shell [[ `id -u` == 0 ]] && echo "-o root")
 
 install:	dvd+rw-tools
-	[[ -d $(prefix)/bin ]] || mkdir -p $(prefix)/bin
-	install $(minus_o) -m $(bin_mode) $(CHAIN) $(prefix)/bin
-	[[ -d $(manprefix)/man1 ]] || mkdir -p $(manprefix)/man1
-	install $(minus_o) -m 0644 growisofs.1 $(manprefix)/man1
+	[[ -d $(DESTDIR)$(prefix)/bin ]] || mkdir -p $(DESTDIR)$(prefix)/bin
+	install $(CHAIN) $(DESTDIR)$(prefix)/bin
+	[[ -d $(DESTDIR)$(manprefix)/man1 ]] || mkdir -p $(DESTDIR)$(manprefix)/man1
+	install growisofs.1 $(DESTDIR)$(manprefix)/man1
+
 	[[ -f rpl8 ]] && \
-	  install $(minus_o) -m $(bin_mode) rpl8 $(prefix)/bin
+	  install rpl8 $(DESTDIR)$(prefix)/bin
 	[[ -f btcflash ]] && \
-	  install $(minus_o) -m $(bin_mode) btcflash $(prefix)/bin
+	  install btcflash $(DESTDIR)$(prefix)/bin
 ])
 
 # common section
