--- dvd+rw-tools-6.0/growisofs.c.orig	2006-01-28 18:06:28.000000000 +0100
+++ dvd+rw-tools-6.0/growisofs.c	2006-01-28 18:07:52.134334250 +0100
@@ -434,7 +434,7 @@
 #endif
 
 #ifndef LLD
-#define LLD		"lld"
+#define LLD		"zd"
 #endif
 
 typedef ssize_t (*pwrite64_t)(int,const void *,size_t,off64_t);
@@ -2768,10 +2768,10 @@
 
 	if (dev_found == 'F') {
 		off64_t capacity = 0;
-		printf("next_session=%lld\n", next_session*CD_BLOCK);
+		printf("next_session=%ld\n", next_session*CD_BLOCK);
 		if (ioctl_handle!=INVALID_HANDLE)
 			capacity = get_capacity (ioctl_handle);
-		printf("capacity=%lld\n", capacity);
+		printf("capacity=%zd\n", capacity);
 		exit(0);
 	}
 
