--- dvd+rw-tools-6.1/growisofs.c.orig	2006-04-07 09:42:51.000000000 +0200
+++ dvd+rw-tools-6.1/growisofs.c	2006-04-07 09:50:33.000000000 +0200
@@ -456,8 +456,12 @@
 #endif
 
 #ifndef LLD
+#if defined(_LP64) || defined(__LP64__)
+#define LLD		"ld"
+#else
 #define LLD		"lld"
 #endif
+#endif
 
 #ifndef DEFAULT_BUF_SIZE_MB
 /* (!)	default is 32MB, which is somewhat wasteful for 4x recording
@@ -2901,10 +2905,10 @@
 
 	if (dev_found == 'F') {
 		off64_t capacity = 0;
-		printf("next_session=%lld\n", next_session*CD_BLOCK);
+		printf("next_session=%"LLD"\n", next_session*CD_BLOCK);
 		if (ioctl_handle!=INVALID_HANDLE)
 			capacity = get_capacity (ioctl_handle);
-		printf("capacity=%lld\n", capacity);
+		printf("capacity=%"LLD"\n", capacity);
 		exit(0);
 	}
 
